xinference/client.py

```diff
@@ -680,6 +713,8 @@ def get_model(self, model_uid: str) -> "ModelHandle":
                 raise ValueError(f"Unrecognized model ability: {desc['model_ability']}")
         elif desc["model_type"] == "embedding":
             return EmbeddingModelHandle(model_ref, self._isolation)
+        elif desc["model_type"] == "multimodal":
```

Comment:
`multimodal` includes text2img, img2text, text2audio, audio2text, etc. 

`images` could be a better name and is also the way OpenAI names their API.

Reply:
Yes, multimodal includes text2img, img2text, text2audio, audio2text, etc. We can add additional methods to the `MultimodalModelHandle`.

Reply:
Fixed. Thanks.